### PR TITLE
I-#285 create post page scrolling issue fix

### LINKED ISSUE #285: Touch Issue on Create Blood Request Post Page
## Bug Report

**Describe the bug:**
On `Create Blood Request` page, labels are not responding on touch

**Severity:**

- [ ] P0 (Critical)
- [ ] P1 (High Priority)
- [x] P2 (Medium Priority)
- [ ] P3 (Low Priority)

## Impact (optional)

Scroll is not smooth on that page.

### PR DESCRIPTION
**Pull request by Rezwanul-Haque**
Fixes #285 

Resolves #
Used when the changes in the pull request indirectly resolve another issue.

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [x] Bugfix
- [ ] Improvement
- [ ] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
scroll issue on create post page issue fix

Provide a brief overview of the changes introduced in this pull request.
We're wrapping the full scroll view with a container which causes a side effect on scrolling.
In this PR I removed the container to fix the issue

## Related Issues/PRs (optional)

Link to the relevant issue(s) or provide a clear description of the problem or feature being addressed.

## How to test

- [ ] Needs regression testing (add QA needed label)

## Checklist

- [x] I have tested these changes locally and they function as expected.
- [ ] I have added or updated necessary documentation to reflect the changes.
- [x] I have followed the project's coding style and guidelines.
- [ ] I have added appropriate test coverage to ensure the changes are functioning correctly.
- [x] I have checked for any potential conflicts with other branches or pull requests.
- [ ] I have updated necessary OpenAPI specs

## Reviewer Request (optional)

If you have any specific requests or suggestions for reviewers, mention them here.

## Screenshots (optional)

If applicable, provide any additional screenshots or GIFs that showcase the changes made in this pull request.

## Demo Link (optional)

If applicable, provide a link to a live demo or deployment where the changes can be reviewed.

## Known Issues (optional)

If there are any known issues or limitations with the changes made in this pull request, describe them here.

## Additional Notes (optional)

Add any additional notes or comments you think are relevant to this pull request.

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

### CODE REVIEW

File: clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx [hunk 1]
```diff
@@ -91,7 +90,7 @@ const CreateBloodRequest = () => {
             />
           </View>
 
-          <View style={styles.fieldSpacing}>
+          <View style={[styles.fieldSpacing]}>
```
Comment (EbrahimAli-RU) [inline clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx:93]: Is there any reason to make `style` an array?

File: clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx [hunk 2]
```diff
@@ -91,7 +90,7 @@ const CreateBloodRequest = () => {
...
             <Dropdown
               label='Select City'
               isRequired={true}
```
Comment (sakibstark11) [inline clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx:96]: wouldn't isRequired be enough

**Review by sakibstark11 — APPROVED** [review]

File: clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx (hunk 1 above)
Comment (Rezwanul-Haque) [reply clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx:93]: @EbrahimAli-RU Bhai when debugging the issue, I added that but forgot to remove...

File: clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx (hunk 2 above)
Comment (Rezwanul-Haque) [reply clients/mobile/src/donationWorkflow/createUpdateDonation/Donation.tsx:96]: @sakibstark11 Bhai I don't have enough context behind this variable usage...

**Pull request merged**